dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -246,6 +246,11 @@ def present_level
       script_level_id: @script_level.id,
       level_id: @level.id)
 
+    @sublevel_callback = milestone_script_level_url(
+      user_id: current_user.try(:id) || 0,
+      script_level_id: @script_level.id,
+      level_id: '') if @level.game.level_group?
```

Comment:
is it still worth doing the one-line version of an if when it takes multiple lines? :)